onnx/serialization.py

```diff
@@ -103,7 +120,8 @@ def deserialize_proto(self, serialized: bytes, proto: _Proto) -> _Proto:
 class _TextProtoSerializer(ProtoSerializer):
     """Serialize and deserialize text proto."""
 
-    supported_formats = ("textproto",)
+    supported_format = "textproto"
+    file_extensions = frozenset({"textproto", "prototxt", "pbtxt"})
```

Comment:
Why not `txt` as well?

Reply:
I think txt is too generic to be assumed storing protobuf data. Also we wouldn’t want to encourage users to save their model in a txt file. A user can always specify the format explicitly if needed